lib/cli.js

```diff
@@ -109,6 +109,8 @@ cli.prototype.parse = function(argv, next) {
       } catch (err) {
         this.errors.push(err)
       }
+    } else {
+      options.css = fs.readFileSync(__dirname + '/../dist/mermaid.css')
```

Comment:
would prefer this use node's [`path.join`](https://nodejs.org/api/path.html#path_path_join_path1_path2) to ensure it remains cross-platform safe. you'll need to `var path = require('path')` at the top of the file, and then this line becomes:

```javascript
options.css = fs.readFileSync(path.join(__dirname, '..', 'dist', 'mermaid.css'))
```